gauge-browser-agent-otel/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "sl-browser-agent-gauge-integration",
+  "description": "Sample project demonstrating usage of Sealights Browser Agent with OTEL flavor and Gauge with taiko tests",
+  "scripts": {
+    "start": "cd calculator-app && httpster",
```

Comment:
httpster should be a devDependency, no?

Reply:
Yeah this script is not valid anymore, I removed it. It's explained in the README to use npx